Source/Core/Core/FifoPlayer/FifoRecordAnalyzer.cpp

```diff
@@ -65,7 +65,14 @@ void FifoRecordAnalyzer::DecodeOpcode(u8* data)
 		break;
 
 	case GX_LOAD_XF_REG:
+	{
```

Comment:
I prefer less indented code, but we should probably try to use equal indents for all the cases in here (at least the ones with curly braces), right?